### DISCUSSION TITLE
index

### DISCUSSION THREAD

**Discussion by giscus** [am009/git-latexdiff-web#1] [Giscus]
# index

A web interface for latexdiff

http://localhost:3001/

**Comment by am009** [comment]
Welcome to latexdiff.cn! If you have any suggestion or question, just create an [issue](https://github.com/am009/git-latexdiff-web/issues/new) or a [discussion](https://github.com/am009/git-latexdiff-web/discussions/new?category=general)!

**Reply by shindeuttunga** [reply]
I am facing an error 
git latexdiff -v --tmpdirprefix /work HEAD^ --latexopt -shell-escape --ignore-latex-errors --main jsen.tex --preamble /work/preamble.tex --no-cleanup --no-view
I believe this error is responsible for no logo being shown and misplacing my title.

I would like to fix this as soon as possible as I am preparing this document as a response to the comments of IEEE reviewers.

**Reply by shindeuttunga** [reply]
Thank you so much in advance :)

**Reply by am009** [reply]
You can download the result diff project. It is a normal latex project with some special macros for show diff text. Then you can change it to fix these issues.

**Comment by MartinHeroux** [comment]
For anyone trying to get the .tar project folder, when you click on 'Download Diff project TAR', you get what appears to be a .pdf file. However, you can go to the 'file', change the file extension from .pdf to .tar, and you will be able to extract the full latex project.

**Reply by am009** [reply]
Thank you for reporting this issue. It has been fixed, and now the downloaded file name will have the correct .tar extension instead of .pdf.
